x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_lifecycle/failure_store/ingestion_rate.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { Streams } from '@kbn/streams-schema';
+import { useTimefilter } from '../../../../hooks/use_timefilter';
+import { useKibana } from '../../../../hooks/use_kibana';
+import type { FailureStoreStats } from '../hooks/use_failure_store_stats';
+import { ChartBarPhasesSeries, ChartBarSeries } from '../common/chart_components';
+import { StreamsAppSearchBar } from '../../../streams_app_search_bar';
+
+export function FailureStoreIngestionRate({
+  definition,
+  stats,
+  isLoadingStats,
+}: {
+  definition: Streams.ingest.all.GetResponse;
+  stats?: FailureStoreStats;
+  isLoadingStats: boolean;
+}) {
+  const { isServerless } = useKibana();
+  const { timeState } = useTimefilter();
+
+  return (
+    <EuiPanel hasShadow={false} hasBorder paddingSize="m" grow={false}>
+      <EuiPanel hasShadow={false} hasBorder={false} paddingSize="s">
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem grow={3}>
+            <EuiText>
+              <h5>
+                {i18n.translate('xpack.streams.failureStoreEnabled.ingestionRatePanel', {
+                  defaultMessage: 'Ingestion rate over time',
```

Comment:
We might want to note in the graph heading that this is the ingestion rate for failed documents, not the overall ingestion rate.  "Failed document ingestion rate over time" possible?